assets/js/translator.js

```diff
@@ -460,6 +462,10 @@ function populateTranslationList() {
         numDstCols = Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS) < (TRANSLATION_LIST_COLUMNS + 1)
             ? Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS)
             : TRANSLATION_LIST_COLUMNS;
+
+    numSrcCols = getSrcNumCols(numSrcCols);
```

Comment:
There's no need to make these functions if they're used just once! I don't see a need to reuse them.